### DISCUSSION TITLE
Adaptive Card version 1.5 TableRow dose not show content of table in Teams

### DISCUSSION THREAD

**Discussion by soseki-angfa** [microsoft/AdaptiveCards#8717] [Q&A]
Hello there,

We have json string as following and send it to our channel in team.

`{"type":"message","attachments":[{"contentType":"application/vnd.microsoft.card.adaptive","content":{"type":"AdaptiveCard","body":[{"type":"TextBlock","size":"large","weight":"Bolder","text":"Result"},{"type":"TextBlock","text":"Notify:【SD】\n\n通知対象者:<at>Notify test@test.jp</at> ,  ","color":"attention","wrap":true},{"type":"Table","gridStyle":"default","firstRowAsHeaders":true,"columns":[{"width":1},{"width":1},{"width":1},{"width":3},{"width":1},{"width":1},{"width":1}],"rows":[{"type":"TableRow","cells":[{"type":"TableCell","items":[{"type":"TextBlock","text":"Child SKU","wrap":true,"weight":"Bolder"}]},{"type":"TableCell","items":[{"type":"TextBlock","text":"SKU","wrap":true,"weight":"Bolder"}]},{"type":"TableCell","items":[{"type":"TextBlock","text":"Brand","wrap":true,"weight":"Bolder"}]},{"type":"TableCell","items":[{"type":"TextBlock","text":"Product","wrap":true,"weight":"Bolder"}]},{"type":"TableCell","items":[{"type":"TextBlock","text":"Lowest","wrap":true,"weight":"Bolder"}]},{"type":"TableCell","items":[{"type":"TextBlock","text":"Token","wrap":true,"weight":"Bolder"}]},{"type":"TableCell","items":[{"type":"TextBlock","text":"point","wrap":true,"weight":"Bolder"}]}],"style":"accent"},{"type":"TableRow","cells":[{"style":"good","items":[{"text":"B08ZSQT1XX","type":"TextBlock","wrap":true}]},{"style":"good","items":[{"text":"SD000000","type":"TextBlock","wrap":true}]},{"style":"good","items":[{"text":"SD ORIGIN","type":"TextBlock","wrap":true}]},{"style":"good","items":[{"text":"200ml","type":"TextBlock","wrap":true}]},{"style":"good","items":[{"text":"1300","type":"TextBlock","wrap":true}]},{"style":"good","items":[{"text":"A19090900909","type":"TextBlock","wrap":true}]},{"style":"good","items":[{"text":"1%","type":"TextBlock","wrap":true}]}]}]}],"$schema":"http://adaptivecards.io/schemas/adaptive-card.json","version":"1.6","msteams":{"width":"Full","entities":[{"type":"mention","text":"<at>Notify testi@test.jp</at>","mentioned":{"id":"test@test.jp","name":"@test"}}]}}}]}`

However, when we switch to New Teams, the const of table is empty as following.
Even we have multiple rows to notify, there is only 1 row as shown.
![image](https://github.com/microsoft/AdaptiveCards/assets/106799197/f94cfabf-b8a5-46c4-9252-9d573248da09)

Moreover, some of us get this empty constent notification even not New Teams. 

I can not find if this is a problem with that json string or teams.
So, could you please give us suggesion for this?

Regards,

**Comment by jwoo-msft** [comment]
@soseki-angfa 

Can you raise this issue to the Teams directly? AdaptiveCards doesn't have control over the Teams App.

https://learn.microsoft.com/en-us/adaptive-cards/resources/partners
